packages/now-next/src/utils.ts

```diff
@@ -354,7 +354,7 @@ export async function getDynamicRoutes(
   dynamicPages: string[],
   isDev?: boolean,
   routesManifest?: RoutesManifest
-): Promise<{ src: string; dest: string }[]> {
+): Promise<{ src: string; dest: string; check: true }[]> {
```

Comment:
```suggestion
): Promise<Source[]> {
```

You can do `import { Source } from `@now/routing-utils` and use it everywhere you expect src/dest like this.

Reply:
Updated